wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -104,9 +105,7 @@ namespace :wagn do
   desc 'insert existing card migrations into schema_migrations_cards to avoid re-migrating'
   task :assume_card_migrations do
     require 'card/migration'
-    Card::CoreMigration.schema_mode do
-      ActiveRecord::Schema.assume_migrated_upto_version Card::Version.schema(:core_cards), Card::CoreMigration.paths
-    end
+    Card::CoreMigration.assume_migrated_upto_version
```

Comment:
No schema stuff in Version now, and moved this logic into the shared class method (new).  This is adapted to the new way we handle the two types in these support classes.  I suppose tests for this would be useful now.